docs/backend/subscribers.md

```diff
@@ -11,8 +11,249 @@ myst:
 
 # Subscribers (event handlers)
 
-```{todo}
-Move content from:
--   https://5.docs.plone.org/external/plone.app.dexterity/docs/advanced/event-handlers.html
--   https://5.docs.plone.org/develop/addons/components/events.html
+The Zope Component Architecture’s zope.event package is used to manage subscribeable events in Plone.
+
+Some of the notable characteristics of the Plone event system are:
+
+- It is simple;
+- Subscriber calling order is not modifiable — you cannot set the order in which event handlers are called;
+- Events cannot be cancelled — all handlers will always get the event;
+- Event handlers cannot have return values;
+- Exceptions raised in an event handler will interrupt the request processing.
+
+## Registering an event handler
+
+Plone events can be scoped:
+
+- Globally (no scope)
+- Per content type
+
+### Example: Register an event-handler on your content type’s creation
+
+In your `.product/your/product/configure.zcml` insert:
+```
+<subscriber
+    for=".interfaces.IMyContentTypeClass
+         zope.lifecycleevent.IObjectCreatedEvent"
+    handler=".your_python_file.your_method"
+    />
+```
+The first line defines to which interface you want to bind the execution of your code, which means here, that the code will only be executed if the object is one of your contenttype’s. If you want this to be interface-agnostic, insert an asterix as a wildcard instead.
+
+The second line defines the event on which this should happen, which is here ‘IObjectCreatedEvent’ – for Archetypes you should use ‘Products.Archetypes.interfaces.IObjectInitializedEvent’ instead. For more available possible events to be used as a trigger, see [Event handlers](https://6.docs.plone.org/backend/subscribers.html#event-handlers)
+
+The third line gives the path to the script that is supposed to be executed.
+
+Create your `.product/your/product/your_python_file.py` and insert:
+```
+def your_method(object, event):
```

Comment:
Its a function and it does not help much, tell the reader what it is: a subscriber
```suggestion
def your_subscriber(object, event):
```